examples/with-next-mint-nft/pages/index.tsx

```diff
@@ -13,7 +13,7 @@ import { abi } from '../contract-abi';
 import FlipCard, { BackCard, FrontCard } from '../components/FlipCard';
 
 const contractConfig = {
-  address: '0x86fbbb1254c39602a7b067d5ae7e5c2bdfd61a30',
+  address: '0x029d4C6e97f54559eE87bc013318d0147f982e6e',
```

Comment:
Did you redeploy an example NFT here @MarcusWentz ?

Reply:
Yes: 
https://goerli.etherscan.io/address/0x029d4C6e97f54559eE87bc013318d0147f982e6e